### ISSUE TITLE
Get Lut by Node Index

### ISSUE THREAD

**Issue by ChristyKail** [pedrolabonia/pydavinci#16]
Hi again
I saw on the DR 17.4.6 release notes that they have _"Added scripting API support to get node count and LUT by node index"_

As usual, no official Resolve documentation of how to do this exactly, but I wondered if you'd looked into this and if you had plans to implement this feature in pydavinci?

My use case would be for doing dailies on multi-LUT shows. We usually create a two-node setup per clip, with CDLs on Node 1, and a LUT or LMT on Node 2. However there is no built-in way of extracting the LUT name from the node and exporting that to an ALE or other metadata file, meaning a lot of manual input and room for mistakes.

**Comment by pedrolabonia** [comment]
Hey Christy!

Thanks for the suggestion. Can you send me a DR 17 project file with that setup and an example LUT you're using? 

I'll see how that's possible through the API. Shouldn't be too hard.

**Comment by ChristyKail** [comment]
Hi Pedro
[Multi_LUT_example.zip](https://github.com/pedrolabonia/pydavinci/files/8694508/Multi_LUT_example.zip)

Here's an example DR project file using built-in luts, and a demo script showing what i'm trying to do.

[This](https://www.reddit.com/r/davinciresolve/comments/tqsda2/davinci_resolve_1746_release_notes/) is the link to the release notes where they meake mention of it, hopefully it's in there somewhere!

**Comment by pedrolabonia** [comment]
Hey Christy. Just implemented it in v0.2.3.

Pushed the update to PyPy. You can update using `pip install pydavinci --upgrade`.

Using your project:

```python
>>> from pydavinci import davinci
>>> resolve = davinci.Resolve()
>>> clip = resolve.active_timeline.items('video', 1)[0]
>>> clip.get_lut(2)
'Sony/SLog3SGamut3.CineToLC-709TypeA.cube'
```
